src/scaict_uwu/libs/language/language_tag_factory.py

```
@@ -0,0 +1,55 @@
+        pass
+
+    def get_tag(self, tag) -> LanguageTag:
+        """
+        Get LanguageTag object by BCP 47 language tag.
+
+        Returns:
+            LanguageTag: The LanguageTag object to the responding BCP 47\
+                language tag.
+        """
+
+        if not tag in self._tags:
```

Ruff (0.8.0)

Test for membership should be `not in`

Convert to `not in`

(E713)